contracts/examples/dao/DAO.sol

```diff
@@ -49,83 +69,127 @@ contract DAO is VentureEth {
     }
 
     /**
-     * @notice Propose funding for venture. Can only be used after the original funding round. venture must be of type IssuanceEth.
-     * @param funding The amount to proposed as funding for the venture.
-     * @param venture The proposed venture. Must be of type IssuanceEth.
+     * @notice Propose a venture. Must have approved the DAO to spend proposalGage of your DAO tokens. venture must inherit from VentureEth.
+     * @param venture The address of the VentureEth contract
      */
-    function proposeVenture(uint256 funding, address venture) public {
-        require(currentState == "LIVE", "DAO needs to be LIVE.");
+    function propose(
+        address venture,
+        uint256 funding
+    ) public onlyWhen("LIVE") {
         uint256 newFundingPool = fundingPool.add(funding);
```

Comment:
I have an idea, let's just not require for proposals to be within the budget. If the DAO stakeholders are silly enough to pass a proposal that can't be funded, that's their problem :man_shrugging: